enforcer-rules/src/main/java/org/apache/maven/plugins/enforcer/RequirePluginVersions.java

```diff
@@ -278,8 +278,9 @@ public void execute( EnforcerRuleHelper helper )
                     {
                         newMsg.append( "SNAPSHOT " );
                     }
-                    newMsg.append( "are not allowed )\n" );
+                    newMsg.append( "are not allowed)" );
                 }
+                newMsg.append( "\n" );
                 for ( Plugin plugin : failures )
```

Comment:
You consistently use `\n` in this file. If you want to switch, that should be a distinct PR. It would fail the requirements for "tiny PRs" that don't need CLAs.

Reply:
@jsoref 
Fix the line as @khmarbaise has proposed.

Reply:
@Tibor17: I'm matching the existing style. Such a change is beyond the scope of this PR, you really shouldn't ask me to do this.
https://github.com/apache/maven-enforcer/blob/d7d33f271865408a8c4207fb591e755ca5876bf9/enforcer-rules/src/main/java/org/apache/maven/plugins/enforcer/RequirePluginVersions.java#L283
https://github.com/apache/maven-enforcer/blob/d7d33f271865408a8c4207fb591e755ca5876bf9/enforcer-rules/src/main/java/org/apache/maven/plugins/enforcer/RequirePluginVersions.java#L312